package.json

```diff
@@ -3,30 +3,52 @@
   "version": "0.0.1",
   "description": "SD-JWT implementation in typescript",
   "source": "src/index.ts",
-  "main": "src/index.js",
   "private": true,
   "scripts": {
     "build": "tsc",
+    "build-all": "npm-run-all clear build:*",
+    "build:browser": "tsc -b ./tsconfig/browser.json && echo '{\"type\": \"module\"}'> dist/browser/package.json",
+    "build:browser-bundle-min": "esbuild --minify --bundle dist/browser/index.js --format=esm --target=es2018 --outfile=dist/browser/index.bundle.min.js",
+    "build:node-cjs": "tsc -b ./tsconfig/node-cjs.json",
+    "build:node-esm": "tsc -b ./tsconfig/node-esm.json && echo '{\"type\": \"module\"}'> dist/node/esm/package.json",
+    "build:types": "tsc -b ./tsconfig/types.json",
+    "clear": "rm -Rf ./dist",
     "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
     "lint:check": "eslint \"{src,test}/**/*.ts\"",
     "test": "jest --verbose"
   },
   "engines": {
-    "node": "18.16.0"
+    "node": ">=18"
   },
   "repository": "@meeco/sd-jwt",
   "keywords": [
     "sd-jwt"
   ],
+  "exports": {
+    ".": {
+      "types": "./dist/types/index.d.ts",
+      "bun": "./dist/browser/index.js",
```

Comment:
@linasi I didn't quite check this, but it was referencing to the browser build so I thought I'd just include it.
But this is quite interesting.
I think it might be referring to this https://bun.sh/ which is a new javascript runtime.

I'll take it off since I don't want to be 'supporting' something we don't quite understand.